### PR TITLE
updated docs link

### PR DESCRIPTION
**Pull request by laurafitzgerald**
**What**
Update docs link to point to redirect

@wei-lee

### CODE REVIEW

File: apb.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ tags:
 metadata:
   displayName: Metrics
   imageUrl: "https://avatars1.githubusercontent.com/u/3380462?s=200&v=4"
-  documentationUrl: "https://docs.aerogear.org/aerogear/latest/metrics/index.html"
+  documentationUrl: "https://docs.aerogear.org/external/apb/metrics.html"
   providerDisplayName: "Red Hat, Inc."
   sdk-docs-android: "https://docs.aerogear.org/aerogear/latest/metrics/index.html"
   sdk-docs-cordova: "https://docs.aerogear.org/aerogear/latest/metrics/index.html"
```
Comment (wei-lee) [inline apb.yml]: what about those?

Also I think we need to run `apb prepare` to regenerate the dockerfile.

**Comment by laurafitzgerald** [comment]
https://github.com/aerogear/docs.aerogear.org/pull/46

File: apb.yml (hunk 1 above)
Comment (laurafitzgerald) [reply apb.yml]: @wei-lee after searching the the origin-web-console fork there is no occurrence of these variables. The are also not included in other apbs. I think we can remove them.

Comment (wei-lee) [reply apb.yml]: @maleck13 can you confirm that those variables can be removed or not?

**Comment by wei-lee** [comment]
I am going to leave the other links here just in case it breaks anything.

**Pull request closed without merging**